tests/utils.js

```diff
@@ -0,0 +1,27 @@
+"use strict";
+
+var utils = {};
+module.exports = utils;
+
+utils.set = function ( key, val )
+{
+    browser.executeScript(
+        "var scope = angular.element( document.body ).injector().get( '$rootScope' );"
+        + "scope['" + key + "'] = JSON.parse( '" + JSON.stringify( val ) + "' );"
+        + "scope.$apply();" );
+};
+
+utils.applyScope = function ()
+{
+    browser.executeScript(
+        "var scope = angular.element( document.body ).injector().get( '$rootScope' );"
+        + "scope.$apply();" );
```

Comment:
really more of a consistency thing with the other 2 here, but I suppose it should be removed anyway